WooCommerce/Classes/Model/OrderStatus+Woo.swift

```diff
@@ -9,6 +9,7 @@ extension OrderStatus {
     /// Returns a collection of all of the known Order Status
     ///
     static var knownStatus: [OrderStatus] {
-        return [.pending, .processing, .onHold, .failed, .cancelled, .completed, .refunded]
+        // .onHold is an existing status but we don't want it to display.
```

Comment:
Perhaps I don't understand the issue here fully, but I don't think we want to remove the `on hold` order status completely from filtering:

![simulator screen shot - iphone xs - 2019-01-22 at 10 36 36](https://user-images.githubusercontent.com/154014/51550326-c2168f80-1e31-11e9-88ac-458a453f166b.png)

I am assuming there is a use case where the customer will want to filter on orders with this status (in the past).

/cc @juliaamosova

Reply:
You are correct. I mis-read some of the conversations in the Android ticket. Thanks for catching that. I'll restore `On Hold` in the filter menu.